modules/utxo-lib/src/address.ts

```diff
@@ -3,11 +3,18 @@ import { Base58CheckResult, Bech32Result } from 'bitcoinjs-lib/src/address';
 
 import * as zcashAddress from '../src/bitgo/zcash/address';
 import { isValidNetwork, isZcash, Network } from './networks';
+import { p2trPayments } from './index';
 
 export function fromOutputScript(outputScript: Buffer, network: Network): string {
   if (isValidNetwork(network) && isZcash(network)) {
     return zcashAddress.fromOutputScript(outputScript, network);
   }
+  try {
+    return p2trPayments.p2tr({ output: outputScript, network }).address as string;
```

Comment:
please repeat the explanation in the commit message here

Reply:
done